### ISSUE TITLE
workspace highlight outline doesn't match theme

### ISSUE THREAD

**Issue by cobalt2727** [pop-os/cosmic-workspaces#30]
I realize this is more of a nitpick than anything else, but when the workspace overview is opened up, the outline of the currently selected workspace is stuck as orange without any way to change this based on the user's preferences - this is new behavior after having upgrade to Pop!_OS 21.10. I understand this blends in fine with the regular Pop light/dark themes, but I'm pretty sure I remember it blending in with custom themes a bit more in 21.04

![image](https://user-images.githubusercontent.com/60624944/147637515-a1d24d68-1ce5-4d53-b542-927b4fde8f84.png)

**Comment by cobalt2727** [comment]
Seems like a result of https://github.com/pop-os/cosmic-workspaces/issues/1, but stuff like this is out of my range of knowledge so I'm not sure

**Comment by cobalt2727** [comment]
I should also note the colored outline surrounding the password entry box on the login screen also has the same issue

**Comment by cobalt2727** [comment]
@13r0ck is this being worked on for 22.04?

**Comment by 13r0ck** [comment]
Not in time for 22.04, but system-wide themeing is very high on the list for the Rust based desktop environement that is planned for release in the Pop!_OS release after 22.04 :smile: 

If you would like to modify the color on your current install you can compile a custom version of cosmic-workspaces with the color you like:

```bash
mkdir custom-cosmic-workspaces
cd custom-cosmic-workspaces
git clone https://github.com/pop-os/cosmic-workspaces
cd cosmic-workspaces
```
Edit [this line](https://github.com/pop-os/cosmic-workspaces/blob/master_impish/stylesheet.css#L93) to the hex value you would like:

Then run the following commands from the `./custom-cosmic-workspaces/cosmic-workspaces` folder:

`sudo echo start && dpkg-buildpackage -us -uc -b && cd .. && sudo dpkg -i *.deb`

Then reboot.

Then your picker indicator should be the color that you are looking for! You are then free to delete the `custom-cosmic-workspaces` folder and all of its contents.

Let me know if you have any issues.

**Comment by cobalt2727** [comment]
wow, thanks for the detailed reply! I'll give that a shot when I get some time and let you know how it goes.

**Comment by cobalt2727** [comment]
@13r0ck Looks great!

![image](https://user-images.githubusercontent.com/60624944/171926966-cf2380ad-f5fe-4ae9-9e57-4e00032023d6.png)

Two minor quirks though:
`dpkg-buildpackage: info: source distribution impish` (might want to update this line)
And when searching for anything, the results are too high up onscreen and overlap with the search box itself, though I'll chalk that up to the extension just not being heavily tested against Gnome 42
![image](https://user-images.githubusercontent.com/60624944/171927520-41bd32df-aef4-41e9-9c2d-1730bba8ea22.png)


edit: this happens with the `gnome-shell-extension-cosmic-workspaces` package from 22.04 repos as well, and happens on both X11 and Wayland (I enabled the latter in a config on my setup months ago)

**Comment by 13r0ck** [comment]
@cobalt2727 
> I'll chalk that up to the extension just not being heavily tested against Gnome 42

Nope, very well tested in gnome 42, this has been released for months now on Gnome 42 and works just tine. ( As long as you built from the `master_jammy` branch ) The issue is your custom extension setup. Gnome has an easily breakable extension system, where one extension's changes can cause breaks absolutely anywhere else in gnome-shell including other extensions. 

For us to build the cosmic appearance in gnome-shell we had to make decisions that can break if all of our extensions are not enabled in the way that the OS is released. ( See https://github.com/pop-os/cosmic-workspaces/issues/35#issuecomment-1095584310 ) Though when released Pop!_OS's cosmic DE will resolve these problems with a completely different extension architecture that will prevent these types of problems.
